modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-customizations.adoc
+// * installing/installing_azure/installing-azure-government-region.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
+// * installing/installing_azure/installing-azure-private.adoc
+// * installing/installing_azure/installing-azure-vnet.adoc
+
+:_content-type: PROCEDURE
+[id="finalizing-encryption_{context}"]
+= Finalizing user-managed encryption after installation
+If you installed {product-title} using a user-managed encryption key, you can complete the installation by updating the default storage class and granting `Owner` permissions to the Azure cluster resource group.
+
+.Procedure
+
+. Update the annotations on the default storage class to remove it as the default. The storageClass that uses the encrypted storage will be the default storageClass instead.
++
+[source,terminal]
+----
+$ oc patch storageclass managed-premium -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
```

Comment:
I think there is missing information here. The "managed-premium" storage class needs to be created here before it can be patched.

Reply:
OK, I didn't see this in [your procedure](https://github.com/jhixson74/installer/blob/60411a2b26d5c21ce8f85b409baf5a0247cc9574/docs/user/azure/encryption.md). What should the contents of the `managed-premium` storage class be?